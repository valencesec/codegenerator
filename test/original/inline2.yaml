listOfItems:
  - thirdItem
  - fourthItem
block:
  aKey: aValue
