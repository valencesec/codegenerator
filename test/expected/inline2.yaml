listOfItems:
  - thirdItem
  - fourthItem
block:
  aKey: aValue
testHas: 123